### PR TITLE
chore(components/pytorch) - Samples fix for PT and PTL Upgrade

### PR DESCRIPTION
**Pull request by jagadeeshi2i**
**Description of your changes:**
Fixes for PT and PTL Upgrades.

**Checklist:**
- [ ] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention).

### CODE REVIEW

File: samples/contrib/pytorch-samples/Pipeline-Cifar10-hpo.ipynb [hunk 1]
```diff
@@ -216,8 +216,7 @@
    "source": [
     "DEPLOY_NAME=\"torchserve\"\n",
     "MODEL_NAME=\"cifar10\"\n",
-    "ISVC_NAME=DEPLOY_NAME+\".\"+NAMESPACE+\".\"+\"example.com\"\n",
-    "INPUT_REQUEST=\"https://raw.githubusercontent.com/kubeflow/pipelines/master/samples/contrib/pytorch-samples/cifar10/input.json\""
```
Comment (chauhang) [inline samples/contrib/pytorch-samples/Pipeline-Cifar10-hpo.ipynb:220]: Don't we need the input for running this notebook?

File: samples/contrib/pytorch-samples/Pipeline-Cifar10.ipynb [hunk 2]
```diff
@@ -209,7 +209,7 @@
     "DEPLOY_NAME=\"torchserve\"\n",
     "MODEL_NAME=\"cifar10\"\n",
     "ISVC_NAME=DEPLOY_NAME+\".\"+NAMESPACE+\".\"+\"example.com\"\n",
-    "INPUT_REQUEST=\"https://raw.githubusercontent.com/kubeflow/pipelines/master/samples/contrib/pytorch-samples/cifar10/input.json\""
+    "INPUT_REQUEST=\"https://raw.githubusercontent.com/kubeflow/pipelines/0abe0f0308d7122d801b1b19dda1fb89fb01f87d/samples/contrib/pytorch-samples/cifar10/input.json\""
```
Comment (chauhang) [inline samples/contrib/pytorch-samples/Pipeline-Cifar10.ipynb:212]: URL is from a commit branch, can we use the one from master branch?

File: samples/contrib/pytorch-samples/cifar10/cifar10_train.py [hunk 3]
```diff
@@ -205,7 +206,3 @@ def training_epoch_end(self, outputs):
         """
         self.show_activations(self.reference_image)
 
-        # Logging graph
-        if self.current_epoch == 0:
-            sample_img = torch.rand((1, 3, 64, 64))
-            self.logger.experiment.add_graph(CIFAR10Classifier(), sample_img)
```
Comment (chauhang) [inline samples/contrib/pytorch-samples/cifar10/cifar10_train.py:211]: Is this not needed anymore?

**Review by chauhang — CHANGES_REQUESTED** [review]
Please see inline comments

File: samples/contrib/pytorch-samples/Pipeline-Cifar10-hpo.ipynb (hunk 1 above)
Comment (jagadeeshi2i) [reply samples/contrib/pytorch-samples/Pipeline-Cifar10-hpo.ipynb:220]: @chauhang The HPO example covers till the training step and those variables were not used.

File: samples/contrib/pytorch-samples/Pipeline-Cifar10.ipynb (hunk 2 above)
Comment (jagadeeshi2i) [reply samples/contrib/pytorch-samples/Pipeline-Cifar10.ipynb:212]: Modified the input request file in this PR. Will update with master branch URL in another PR after merge.

File: samples/contrib/pytorch-samples/cifar10/cifar10_train.py (hunk 3 above)
Comment (jagadeeshi2i) [reply samples/contrib/pytorch-samples/cifar10/cifar10_train.py:211]: Yes @chauhang This is handled by tensorboard - https://pytorch-lightning.readthedocs.io/en/stable/api/pytorch_lightning.loggers.tensorboard.html

**Review by chauhang — APPROVED** [review]

**Comment by chauhang** [comment]
lgtm

**Pull request merged**